### PR TITLE
Update ppp

### LINKED ISSUE #361: Update PPP sean

### PR DESCRIPTION
**Pull request by seanjyjy**

### CODE REVIEW

File: src/test/java/seedu/internhunter/logic/commands/util/CommandUtilTest.java [hunk 1]
```diff
@@ -161,4 +163,19 @@ public void getProfile_validIndex_noExceptionThrown() {
         assertDoesNotThrow(() -> getProfileItem(modelWithData, Index.fromOneBased(1)));
         assertDoesNotThrow(() -> getProfileItem(modelWithData, Index.fromZeroBased(0)));
     }
+
+    @Test
+    public void getFullListIndex_equals_success() {
+        ProfileItem profileItem = profileItemBuilder.withTitle("hello world").withDescriptors("ok").build();
+        modelWithData.addProfileItem(profileItem);
+        ApplicationItem applicationItem = applicationItemBuilder.withInternshipItem(LAZADA_DS)
```
Comment (orzymandias) [inline src/test/java/seedu/internhunter/logic/commands/util/CommandUtilTest.java:175]: I noticed for some fields you use the predefined fields like LAZADA_DS but some you used random strings, maybe stick to using one?

File: src/main/java/seedu/internhunter/model/Model.java [hunk 2]
```diff
@@ -38,7 +38,7 @@
 
     /**
      * Returns the company list
```
Comment (orzymandias) [inline src/main/java/seedu/internhunter/model/Model.java]: Missing full stop

Comment (orzymandias) [reply src/main/java/seedu/internhunter/model/Model.java]: Same for the other getXXXList

**Review by orzymandias — COMMENTED** [review]
LGTM, just some minors nits and it will be good to merge

File: src/main/java/seedu/internhunter/model/Model.java (hunk 2 above)
Comment (seanjyjy) [reply src/main/java/seedu/internhunter/model/Model.java]: fixed

File: src/test/java/seedu/internhunter/logic/commands/util/CommandUtilTest.java (hunk 1 above)
Comment (seanjyjy) [reply src/test/java/seedu/internhunter/logic/commands/util/CommandUtilTest.java:175]: fixed

**Review by orzymandias — APPROVED** [review]

**Pull request merged**